addons/mail/static/src/models/message_composition/message_composition.js

```diff
@@ -0,0 +1,212 @@
+                body = body.replace(text, placeholder);
+            }
+            const baseHREF = this.env.session.url('/web');
+            for (const mention of mentions) {
+                const href = `href='${baseHREF}#model=${mention.model}&id=${mention.id}'`;
+                const attClass = `class='${mention.class}'`;
+                const dataOeId = `data-oe-id='${mention.id}'`;
+                const dataOeModel = `data-oe-model='${mention.model}'`;
+                const target = `target='_blank'`;
+                const link = `<a ${href} ${attClass} ${dataOeId} ${dataOeModel} ${target}>${mention.text}</a>`;
+                body = body.replace(mention.placeholder, link);
+            }
+            return body;
+        },
+        /**
+         * @private
+         * @param {string} htmlString
+         * @returns {string}
+         */
+        _generateEmojisOnHtml(htmlString) {
+            for (const emoji of emojis) {
+                for (const source of emoji.sources) {
+                    const escapedSource = String(source).replace(
+                        /([.*+?=^!:${}()|[\]/\\])/g,
+                        '\\$1');
+                    const regexp = new RegExp(
+                        '(\\s|^)(' + escapedSource + ')(?=\\s|$)',
+                        'g');
+                    htmlString = htmlString.replace(regexp, '$1' + emoji.unicode);
+                }
+            }
+            return htmlString;
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsLog() {
+            return this.composer.isLog;
+        },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeThread() {
+            return replace(this.composer.thread);
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeRawBody() {
+            return this.composer.textInputContent;
+        }
+    },
+    fields: {
+        /**
+         * States which attachments are currently being created in this message composition.
+         */
+        attachments: many2many('Attachment', {
+            inverse: 'messageCompositions',
+        }),
+        /**
+         * The body of message resulting from this message composition.
+         * It's created when needed to avoid performance issue.
+         */
+        body: attr(),
+        composer: one2one('Composer', {
+            inverse: 'messageComposition',
+            readonly: true,
+            required: true,
+        }),
+        /**
+         * States the body without any transformation. It basically the content of
+         * the composer.
+         */
+        rawBody: attr({
+            compute: '_computeRawBody',
+        }),
+        /**
+         * States the mentionned channel in the rawBody.
+         */
+        mentionedChannels: many2many('Thread'),
+        /**
+         * States the mentionned partners in the rawBody.
```

Comment:
Just "body" (instead of "rawBody") is good.
"rawBody" gives the impression these fields are more technical that they really are.

Reply:
Done !